CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 # Unreleased
 
 - Highlight token aliases in Menhir associativity declarations (#473)
+- The extension is not activated when an OCaml project is detected (#482)
```

Comment:
If we're describing a bug fix, we should allude to that with something like

> Fix activation of extension. The extension was not activated when an OCaml project was detected.

Or we can simply describe the new behavior:

> Activate the extension when workspace contains OCam, Reason sources or project marker files.

Reply:
Meant to write "now":
"The extension is now activated when an OCaml project is detected"

Reply:
But your version is more precise, will replace it :)